### DISCUSSION TITLE
Can we use services and tools other than Hasura?

### DISCUSSION THREAD

**Discussion by PtradeLLC** [nhost/nhost#2277] [Q&A]
Our app is an existing NextJs app that we are trying to integrate Nhost with. Our company uses tools such as `Prisma` as `ORM` to talk to cloud hosted `PostgreSQL` database.  Can we use other tools and services such as the ones mentioned with Nhost or it only work with `Hasura / GraphQL` ?

Apologies for the novice question as I am just learning about Nhost.

Thanks,

**Comment by dbarrosop** [comment]
Yes, you can:

https://docs.nhost.io/run

Nhost Run allows you to run any workload you can containerize.
